debian/kolibri.templates

```diff
@@ -34,3 +34,14 @@ Description: Running Kolibri manually:
  Start Kolibri when the system boots:
  .
    sudo systemctl enable kolibri
+
+Template: kolibri/user
+Type: string
+Default: kolibri
+Description: Which user account should own the Kolibri server?
+ The default is to choose your preferred desktop user account, to have easier
+ access to Kolibri's data without having to change privileges.
```

Comment:
Maybe

> The default is to choose your preferred desktop user account. This simplifies system permissions and data storage.

Reply:
Ya, true -- I understand that you mean _maybe we detect a desktop user and use that as the default_? I was a bit unsure what to put in this string because it's static, so has to give good guidance regardless of whether the default value is the actual desktop username or "kolibri".

Reply:
Note that the "kolibri" default value above is the fallback when we fail to overwrite it with a desktop user.

Reply:
the other motivation for my proposed change was that "to have easier access to Kolibri's data without having to change privileges" is just one benefit. Another important one is that Kolibri can access USB drives easier. It's easier just to be somewhat vague.

Reply:
Thanks I've tried to address it more vaguely by adding a "for instance" and using USB devices as the example:

```
Description: Which user account should own the Kolibri server?
 The default is to choose your preferred desktop user account, for instance
 to ensure access to importing data from external USB devices.
 .
 Entering a username that doesn't exist will create a new system user with
 home directory /var/<username>.
```